components/SendMoneyToCollectiveBtn.js

```diff
@@ -92,22 +103,22 @@ class SendMoneyToCollectiveBtn extends React.Component {
   }
 }
 
-const paymentMethodsQuery = gql`
+const paymentMethodsQuery = gqlV2/* GraphQL */ `
   query SendMoneyToCollectivePaymentMethods($slug: String) {
-    Collective(slug: $slug) {
+    account(slug: $slug) {
       id
-      paymentMethods(service: "opencollective") {
+      paymentMethods(types: ["collective"]) {
```

Comment:
No!

`service: "opencollective"` AND `type: "collective"`

Reply:
@znarf : You meant, `service: "opencollective" and `types: ["collective"]` right? I just changed to code to have both of these arguments. Now I think it should be good. 🙂